list/list.go

```diff
@@ -878,6 +922,12 @@ func (m *Model) handleFiltering(msg tea.Msg) tea.Cmd {
 			if m.FilterInput.Value() == "" {
 				m.resetFiltering()
 			}
+		case key.Matches(msg, m.KeyMap.AcceptWhileAdding):
+			m.hideStatusMessage()
+			m.FilterInput.Blur()
+			m.filterState = Unfiltered
+			m.updateKeybindings()
+			return func() tea.Msg { return ItemAddedMsg(m.FilterInput.Value()) }
```

Comment:
Small note here: logically speaking, there's no need to send a message for internal app communication. That is, commands are strictly for IO.